README.md

```diff
@@ -1,3 +1,32 @@
 # CivicTechExchange
 
-.
+Linux setup:
+
+# install postgress and venv
+sudo apt-get update
+sudo apt-get install postgresql postgresql-contrib
+sudo apt-get install python3-venv
+
+# checkout the repo
+git clone https://github.com/DemocracyLab/CivicTechExchange.git
+
+# create and activate venv
+cd CivicTechExchange/
+python -m venv myenv
+source myvenv/bin/activate
+
+# install python libraries
+pip install --upgrade pip
+pip install django
+pip install dj_database_url
+pip install djangorestframework
+pip install taggit
+pip install django-taggit
+pip install whitenoise
+pip install psycopg2
+pip install boto3
+
+# create database via django and start test server
```

Comment:
Need to include node/yarn steps as well.  

Installing Node.js:
curl -sL https://deb.nodesource.com/setup_8.x | sudo -E bash -
sudo apt-get install nodejs

Installing Yarn package manager:
curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
sudo apt-get update && sudo apt-get install yarn

Building/Collecting static assets before runserver:
npm run build